README.md

```diff
@@ -35,6 +35,14 @@ command_bus.register(FooCommand, -> (foo_cmd) { FooService.new(event_store: even
 command_bus.register(BarCommand, -> (bar_cmd) { BarService.new.call(bar_cmd) })
 ```
 
+Alternatively, you can register a Class and it will be instantiated per call, which provides the same behavior as `subscribe` in RailsEventStore.
```

Comment:
I am not sure we should mention RES here.

Reply:
Agree. Addressed in most recent commit.